README.md

```diff
@@ -14,149 +14,3 @@ Note that the README in this repo is targeted at Calico docs contributors.
 
 **If you are looking for the repository formerly known as `projectcalico/calico`,
 it has been renamed to [`projectcalico/felix`](https://github.com/projectcalico/felix).**
-
```

Comment:
I think you should update the title of this page from "Project Calico Documentation" to simply "Calico"